junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -1057,6 +1057,41 @@ public static boolean isMethodPresent(Class<?> clazz, Predicate<Method> predicat
 		// @formatter:on
 	}
 
+	/**
+	 * Get the values of the specified fields on the given instance.
+	 *
+	 * @param fields list of fields on a class; never {@code null}
+	 * @param object instance of a class; never {@code null}
+	 * @return an immutable list of values of the specified fields on the given instance
+	 */
+	public static List<Object> getFieldValuesForInstance(List<Field> fields, Object object) {
```

Comment:
No need to say "for instance": we should rather support retrieval of static and non-static fields with the same API and document that the instance can be `null` when retrieving static fields as we do elsewhere.

Reply:
removed for instance from name